app/src/main/java/de/danoeh/antennapod/fragment/SubscriptionFragment.java

```diff
@@ -376,7 +398,7 @@ public void onEventMainThread(DownloadEvent event) {
     private final MenuItemUtils.UpdateRefreshMenuItemChecker updateRefreshMenuItemChecker =
             () -> DownloadService.isRunning && DownloadRequester.getInstance().isDownloadingFeeds();
 
-    private final SubscriptionsAdapter.ItemAccess itemAccess = new SubscriptionsAdapter.ItemAccess() {
+    private final SubscriptionsRecyclerAdapter.ItemAccess itemAccess = new SubscriptionsRecyclerAdapter.ItemAccess() {
```

Comment:
To be honest, I don't really get why there is this ItemAccess indirection. I know that it was already there before but if you are touching it anyways, I think it would be great to just get rid of it. Just add a member to the adapter that holds the list directly.

Reply:
Yeah, I kind of had the same thought while using it. I didn't want to introduce too many inconsistencies. I will remove it soon.